### DISCUSSION TITLE
$LOCAL_IP 面板 ip 修改问题，端口不监听无法访问

### DISCUSSION THREAD

**Discussion by kikyo-tcm** [1Panel-dev/1Panel#5182] [Q&A]
### 联系方式

642592769@qq.com

### 1Panel 版本

v1.10.9-lts

### 问题描述

![image](https://github.com/1Panel-dev/1Panel/assets/104256888/1f754f29-a04b-4c58-b1b0-ed55dc46edb8)
端口也在没有监听中
![image](https://github.com/1Panel-dev/1Panel/assets/104256888/e7b42fa9-0521-4ec0-a9e2-18c9a50264b3)

### 重现步骤

初次安装后就如下：
![image](https://github.com/1Panel-dev/1Panel/assets/104256888/6ab4e8ef-835d-4d36-9fa8-9d78c1145e34)

**Comment by wanghe-fit2cloud** [comment]
`1pctl status` 和 `journalctl -u 1panel -exf` 两个命令查看一下服务状态和日志。
